ddsketch/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
```

Comment:
Since this is a new crate, the [CODEOWNERS](https://github.com/DataDog/libdatadog/blob/main/.github/CODEOWNERS) file should probably be updated as well. I assume @DataDog/libdatadog-telemetry should own it.

Reply:
I think both @Datadog/libdatadog-apm and @DataDog/libdatadog-telemetry  since it's going to be used for stats computation. This should require a review from at least one of the two groups, not both looking at [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file).